### PR TITLE
Awesome Books

### PR DESCRIPTION
**Pull request by m00h0**
- Finish the Html flie.
- Finish the CSS file
- Finish the JS file
- Finish the Readme.md file.
- Add the MIT license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,149 @@
...
+
+
+### Key Features <a name="Awesome Books"></a>
+
+
+- **[Awesome Books]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (t-yanick) [inline README.md]: - [ ] Great job working on this milestone. At the moment, your `live demo` link redirects to your github repo. Deployment  is not a requirement for this milestone. If you can not get the correct link, kindly delete this subsection.

File: README.md [hunk 2]
```diff
@@ -1 +1,149 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse team.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 📝 License <a name="license"></a>
+
+This project is [MIT](MIT.md) licensed.
```
Comment (t-yanick) [inline README.md:138]: - [ ] Great job working on this milestone and updating the ReadMe. Unfortunately, when I click on the MIT license link, it gives me an error 404 page. It is not professional to allow broken links in our codebase. Kindly fix this issue.
image

File: README.md [hunk 3]
```diff
@@ -1 +1,149 @@
...
+- [ ] **[Made Mobile project]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="Microverse"></a>
+
+👤 **Bianca**
+
+- GitHub: [@githubhandle](https://github.com/BiancaMe)
```
Comment (t-yanick) [inline README.md]: - [ ] Kindly take this information to the author section.

Comment (t-yanick) [inline README.md]: - [ ] Kindly take this information to the author section.

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi Team, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  App works as required (Adding and removing books)
:heavy_check_mark: Data is preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md)
:heavy_check_mark: Use of array methods that don't mutate the array
:heavy_check_mark: Use of correct Github flow

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1 +1,140 @@
...
+    <li><a href="https://microverse.org/">Microverse</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="Awesome Books"></a>
+
+
+- **[Awesome Books]**
+
```
Comment (AbelG101) [inline README.md:52]: - [ ] [OPITONAL] As a developer, it's important to not only write clean and well-organized code but also to document your work effectively. One key aspect of this is creating a detailed and informative README file. In particular, the Future Features & Key Features section of your README presents a great opportunity to showcase your creativity and vision for the project. I would highly recommend adding a compelling and descriptive feature to this section to grab the attention of recruiters or future clients. 😊

**Review by AbelG101 — APPROVED** [review]
# Hi @newtonah0 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Add & Remove functionality is properly implemented ✔
- Books are properly displayed ✔
- Correct use of GitHub Flow ✔
- All linter checks are passing ✔
- Descriptive PR title with a good summary ✔

## Optional suggestions
- Check the comments below...

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**